components/formats-gpl/src/loci/formats/in/ND2Handler.java

```diff
@@ -493,7 +493,7 @@ else if (qName.equals("uiMaxDst")) {
           LOGGER.warn("Could not set the pixel type", e);
         }
       }
-      else if (qName.equals("uiWidthBytes") || qName.startsWith("uiBpc")) {
+      else if (qName.equals("uiWidthBytes") || qName.startsWith("uiBpcInMemory") || qName.startsWith("uiBpcSignificant")) {
```

Comment:
My reluctance with prioritising them is that in this case the "uiBpc" value is incorrect for bitsPerPixel and PixelType and cant be used for this purpose no matter the order of the tags.

For `uiBpcSignificant`, it is used by `parseKeyAndValue` to set the bitsPerPixel in the core metadata. If both values are set then they should set the same PixelType though one will overwrite the other. Im unsure though if we can rely on both being set as there appear to be samples with only one or the other set in which case they would use either for PixelType.